ddtrace/contrib/asyncio/provider.py

```diff
@@ -14,8 +14,28 @@ class AsyncioContextProvider(DefaultContextProvider):
     execution. It must be used in asynchronous programming that relies
     in the built-in ``asyncio`` library. Framework instrumentation that
     is built on top of the ``asyncio`` library, can use this provider.
+
+    This Context Provider inherits from ``DefaultContextProvider`` because
+    it uses a thread-local storage when the ``Context`` is propagated to
+    a different thread, than the one that is running the async loop.
     """
-    def __call__(self, loop=None):
+    def activate(self, context, loop=None):
```

Comment:
Is it normal that this context API change (adding the `activate `) is part of this PR?
Could we merge the context API change to simplify this PR a bit?

cc @palazzem

Reply:
merged the other one so this should not be a problem anymore